src/pages/Quiz/QuestionWrapper/index.js

```diff
@@ -0,0 +1,164 @@
+import React, { useEffect, useState } from 'react';
+
+import { Typography, Paper, Divider, Button, Box } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import Question from '../../../components/Question';
+
+import Score from '../../../components/Score';
+
+import {
+  fetchSelectedQuestions,
+  fetchTestDetails,
+} from '../../../actions/quiz.js';
```

Comment:
You can avoid using `.js` at the end of the file.
All of JS modules (files) can be imported without `.js` extension at the end.

fe:

```js
import {
  fetchSelectedQuestions,
  fetchTestDetails,
} from '../../../actions/quiz';
```

Reply:
Update in next commit